### PR TITLE
Added time left for recommended sensor cleaning

### PR DESCRIPTION
**Pull request by bbbenji**
Unfortunately I am unable to test these changes, however I believe they should work.

### CODE REVIEW

File: miio/vacuumcontainers.py [hunk 1]
```diff
@@ -321,6 +322,10 @@ def filter_left(self) -> timedelta:
     def sensor_dirty(self) -> timedelta:
         return pretty_seconds(self.data["sensor_dirty_time"])
 
+    @property
+    def sensor_dirty_left(self) -> timedelta:
+        return self.side_brush_total - self.sensor_dirty
+
```
[the remark this answers is not in the document]
Comment (bbbenji) [reply miio/vacuumcontainers.py]: https://github.com/rytilahti/python-miio/pull/93

:)

[the remark this answers is not in the document]
Comment (bbbenji) [reply miio/vacuumcontainers.py]: https://github.com/rytilahti/python-miio/pull/119

:)

File: miio/vacuumcontainers.py [hunk 2]
```diff
@@ -292,6 +292,7 @@ def __init__(self, data: Dict[str, Any]) -> None:
         self.main_brush_total = timedelta(hours=300)
         self.side_brush_total = timedelta(hours=200)
         self.filter_total = timedelta(hours=150)
+        self.sensors_total = timedelta(hours=30)
```
Comment (rytilahti) [inline miio/vacuumcontainers.py]: Use singular here, as property is "sensor_dirty". Or if you thing it's worth renaming it now, maybe introduce a new, renamed property and `@deprecate` the old one.

File: miio/vacuumcontainers.py [hunk 3]
```diff
@@ -321,6 +322,10 @@ def filter_left(self) -> timedelta:
     def sensor_dirty(self) -> timedelta:
         return pretty_seconds(self.data["sensor_dirty_time"])
 
+    @property
+    def sensor_dirty_left(self) -> timedelta:
+        return self.side_brush_total - self.sensor_dirty
```
Comment (rytilahti) [inline miio/vacuumcontainers.py]: `side_brush_total` is wrong.

File: miio/vacuumcontainers.py (hunk 2 above)
Comment (bbbenji) [reply miio/vacuumcontainers.py]: Thanks for taking the time to have a look. I have fixed the issues.

File: miio/vacuumcontainers.py [hunk 4]
```diff
@@ -324,7 +324,7 @@ def sensor_dirty(self) -> timedelta:
 
     @property
     def sensor_dirty_left(self) -> timedelta:
-        return self.side_brush_total - self.sensor_dirty
+        return self.sensor_dirty_total - self.sensor_dirty
```
Comment (rytilahti) [inline miio/vacuumcontainers.py]: This is still wrong :-) Also, please update `vacuum_cli.py` accordingly to display the new property.

**Comment by bbbenji** [comment]
I am far from a programmer. Hope this is everything done correctly:) Thanks for your patience.

**Comment by rytilahti** [comment]
Seems to work fine, I just tested it locally :-)
```
Sensor dirty: 1 day, 14:42:18 (left -1 day, 15:17:42)
```
(cleaning the sensors / reseting the status is overdue, it seems..)

**Pull request merged**